src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerExtensionsTests.cs

```diff
@@ -356,6 +357,48 @@ public void AddConsole_NullFormatterNameUsingSystemdFormat_AnyDeprecatedProperti
             Assert.Equal("HH:mm:ss ", formatter.FormatterOptions.TimestampFormat);
         }
 
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [InlineData(-1)]
+        [InlineData(0)]
+        public void AddConsole_MaxQueueLengthSetToNegativeOrZero_IgnoresValue(int invalidMaxQueueLength)
```

Comment:
```suggestion
        public void AddConsole_MaxQueueLengthSetToNegativeOrZero_Throws(int invalidMaxQueueLength)
```

It'd probably be easier to test this by calling `consoleLoggerOptions.MaxQueueLength = invalidMaxQueueLength;` directly, but I can appreciate verifying the end to end. We should probably still verify we're getting the right inner exception though.